lib/config.js

```diff
@@ -227,3 +242,32 @@ config.setLocals = (locals) => {
     locals.devMode = config.devMode;
     locals.is_administrator = false;
 };
+
+/* Some helper functions to access the 'config' table in the database */
+config.getDBConfigValueAsync = async function(key, defaultVal = null) {
+    const result = await sqldb.call('config_select', {key});
+    if (result.rows.length == 0) {
+        return defaultVal;
+    } else {
+        return result.rows[0].value;
+    }
+};
+config.getDBConfigValue = function(key, callback) {
```

Comment:
This function doesn't seem to be quite the sync version of the async function above?